src/core/core.cpp

```diff
@@ -1012,6 +1012,23 @@ void Core::loadFriends()
     delete[] ids;
 }
 
+void Core::loadGroups()
+{
+    QMutexLocker ml{coreLoopLock.get()};
+
+    const size_t groupCnt = tox_conference_get_chatlist_size(tox.get());
```

Comment:
Come on. `Cnt` instead of `Count`. It's not the best idea :)

Reply:
Done